src/vendor/prism/includeLangs.js

```diff
@@ -1,38 +1,38 @@
-// @flow
-
-// These are the languages that'll be included in the generated
-// prism/index.js file
-
-module.exports = {
-  markup: true,
-  bash: true,
-  clike: true,
-  c: true,
-  cpp: true,
-  css: true,
-  "css-extras": true,
-  javascript: true,
-  jsx: true,
-  "js-extras": true,
-  coffeescript: true,
-  diff: true,
-  git: true,
-  go: true,
-  graphql: true,
-  handlebars: true,
-  json: true,
-  less: true,
-  makefile: true,
-  markdown: true,
-  objectivec: true,
-  ocaml: true,
-  python: true,
-  reason: true,
-  sass: true,
-  scss: true,
-  sql: true,
-  stylus: true,
-  typescript: true,
-  wasm: true,
-  yaml: true
-};
+// @flow
+
+// These are the languages that'll be included in the generated
+// prism/index.js file
+
+export default {
```

Comment:
I wouldn't actually convert this, since it's compiled away using the macro, maybe we should just inline it into the macro, so it's clear that this isn't needed?

Reply:
It’s imported elsewhere in real code I believe

Reply:
It’s imported elsewhere in real code I believe

Reply:
Yep, here it is: https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/types.js#L4

```js
import includedLangs from "./vendor/prism/includeLangs";
```

and we _do_ currently transpile `types.js` to `es/`. The fact it's included in our real `es/` output **and** needs to be directly CJS imported in codegen is why I used `esm` (which is the usual tool here).

So, if we can just _not_ output to `es/` maybe as I don't see that `types.js` is actually needed there from what I can tell, then we can simplify things.